src/flag_gems/utils/pointwise_dynamic.py

```diff
@@ -392,9 +397,18 @@ def generate_destination_passing_pointwise_wrapper(
         if rank > 0:
             code.writeline("# strides of each tensor argument w.r.t the task space")
             for i in range(op_desc.num_input_tensors()):
-                code.writeline(
-                    f"in{i}_strides = broadcasted_stride(in{i}.shape, in{i}.stride(), shape)"
-                )
+                code.writeline(f"if 'in{i}_shape' in kwargs:")
+                with code.indent():
+                    code.writeline(f"in{i}_strides = in{i}.stride()")
+                    code.writeline(f"in{i}_shape = kwargs['in{i}_shape']")
+                code.writeline("else:")
+                with code.indent():
+                    code.writeline(
+                        f"in{i}_strides = broadcasted_stride(in{i}.shape, in{i}.stride(), shape)"
+                    )
+                    code.writeline(
+                        f"in{i}_shape = [(num_tasks + 1) for _ in range(len(shape))]"
+                    )
```

Comment:
Why use `in{i}_shape = [(num_tasks + 1) for _ in range(len(shape))]` as in shape?

Reply:
In tile op, `in_shape[i]` will be used when `tl.load`. 
Specifically when rank == 2, change `in0 = tl.load(in0_ptr + i0 * in0_stride0 + i1 * in0_stride1, mask=mask)` into `in0 = tl.load(in0_ptr + (i0 % in0.shape[0]) * in0_stride0 + (i1 % in0_stride0) * in0_stride1, mask=mask)`. 

But for other ops, we don't need to `% in0.shape[0]`. To make sure this change will not affect other ops, `(num_tasks + 1)` is set.